piksi_tools/console/settings.yaml

```diff
@@ -2242,7 +2242,7 @@
   readonly: false
   units: N/A
   Description: Enables or disables the Over-The-Air upgrade daemon's verbose output.
-  Notes: 'ota enabled' must be 'False' in order to change this setting.
+  Notes: The OTA daemon must be disabled in order to change this setting.
```

Comment:
My yaml parser didn't like the `'`s

Reply:
This wasn't about `libsettings`. It was about my environment. Anyway there was also another reason to do this change; to unify the note with the next setting note that has the same semantic.

Reply:
👍